e2e/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "baseUrl": "..", // This must be specified if "paths" is.
     "paths": {
       "@app/*": ["app/*"] // This mapping is relative to "baseUrl".
-    }
+    },
+    "noEmit": true
```

Comment:
Don't need to transpile the code, just check it

Reply:
Nice!